src/contracts/security/security.service.ts

```diff
@@ -109,39 +154,60 @@ export class SecurityService {
   }
 
   /**
-   * Signs a message to pause deposits with the prefix from the contract
+   * Signs a message to pause deposit contract with the prefix from the contract
    */
-  public async signPauseData(
-    blockNumber: number,
-    stakingModuleId: number,
-  ): Promise<Signature> {
+  public async signPauseDataV3(blockNumber: number): Promise<Signature> {
     const prefix = await this.repositoryService.getPauseMessagePrefix();
 
-    return await this.walletService.signPauseData({
+    return await this.walletService.signPauseDataV3({
       prefix,
       blockNumber,
-      stakingModuleId,
     });
   }
 
   /**
-   * Returns the current state of deposits
+   * Sends a transaction to pause deposit contract
+   * @param blockNumber - the block number for which the message is signed
+   * @param signature - message signature
    */
-  public async isDepositsPaused(
+  @OneAtTime()
+  public async pauseDepositsV3(
+    blockNumber: number,
+    signature: Signature,
+  ): Promise<ContractReceipt | void> {
+    this.logger.warn('Try to pause deposits');
+    this.pauseAttempts.inc();
+
+    const contract = await this.getContractWithSigner();
```

Comment:
awaits everywhere seems useless, maybe let's add them only if they do some requests inside?

Reply:
right, here it is useless, will fix